tests/basic/test-geometry-intersection.cpp

```diff
@@ -274,6 +274,13 @@ void test_circle_plane_intersection()
                         || ( are_almost_equal( results3[1], answer32 ) ) ),
         "Test circle crossing the plane (intersection coordinates)" );
 
+    // Circle not crossing the plane (but not parallel)
+    Geometry::Circle circle4{ { { -2., -1., 1. }, { 3., 0., 3. } }, 2. };
+    bool does_circle4_intersect_plane;
+    std::tie( does_circle4_intersect_plane, std::ignore ) =
+        Intersection::circle_plane( circle4, plane );
+    verdict( !does_circle4_intersect_plane, "Test circle distance from plane" );
```

Comment:
Is this comment true: "Test circle distance from plane"? I am not sure about "distance"

Reply:
You are right. I am going to clarify this comment.